react-native/update-font-icon.js

```diff
@@ -0,0 +1,44 @@
+// Generate our react-native icons from the fonticon stylesheet and images
+import fs from 'fs'
+import path from 'path'
+
+const css = fs.readFileSync('../desktop/renderer/fonticon.css', {encoding: 'utf8'})
+const stream = fs.createWriteStream('../shared/common-adapters/icon.paths.native.js')
+
+stream.once('open', () => {
+  stream.write('// This file is generated by update-font-icon.js, DO NOT HAND EDIT!\n')
+  stream.write('// @flow\n')
+  stream.write('\n')
+
+  stream.write('export const fontIcons = {\n')
+
+  const glyphs = []
+  let glyph = null
+  css.split('\n').forEach(line => {
+    if (glyph) {
+      const val = line.match(/content:\ \"\\(.*)\";/)[1]
```

Comment:
If we mess up here, the icons either won't load or be wrong, right? I'm a little worried something non obvious may happen, like one icon uses another icon's image, and the other icon is broken. We may not notice it for a while depending on the icon.